esrally/driver/runner.py

```diff
@@ -501,6 +504,18 @@ async def __call__(self, es, params):
             bulk_params["timeout"] = params["timeout"]
         if "pipeline" in params:
             bulk_params["pipeline"] = params["pipeline"]
+        if "refresh" in params:
+            refresh_params = {"sync": "wait_for", "async": "true", "default": "false"}
+            bulk_params["refresh"] = refresh_params.get(params["refresh"])
+
+            if bulk_params["refresh"] is None:
+                self.logger.warning(
+                    "Using default bulk refresh parameter value '%s' in place of unrecognized specfified parameter value '%s'. "
+                    "Use one of %s.",
+                    refresh_params["default"],
+                    params["refresh"],
+                    ", ".join(refresh_params.keys()),
+                )
```

Comment:
I don't think this needs to be a warning, maybe just an info message.

Reply:
> Edit: I'm not actually sure what we're checking here? If the param isn't set shouldn't we just default to default?

This check is for invalid values, like the test [here](https://github.com/elastic/rally/pull/1743/files#diff-026a25af4075b17db2736adf14aa844c0aaeb2bc995b4b4ae94c432b0380b51fR1365).